.github/workflows/ci.yml

```diff
@@ -91,31 +95,23 @@ jobs:
             type=registry,ref=${{ steps.login.outputs.registry }}/firezone/${{ matrix.image_name }}-buildcache:main
           # This will write the cache on main even if integration tests fail,
           # but it'll just be corrected on the next successful build.
-          cache-to:
-            type=registry,ref=${{ steps.login.outputs.registry }}/firezone/${{
-            matrix.image_name }}-buildcache:${{ env.CACHE_TAG }}
+          cache-to: >-
+            type=registry,ref=${{ steps.login.outputs.registry }}/firezone/${{ matrix.image_name }}-buildcache:${{ env.CACHE_TAG }}
```

Comment:
Can we please come up with a more clean naming convention? Eg. `caches/${{ matrix.image_name }}`. This will make navigating the artifacts list a bit easier.

Reply:
I think this is actually the image name the cache gets pushed to, so I think we need the registry image?

Reply:
Nope, it will work just fine. I'll update it once you done with all the changes.